awx_collection/plugins/doc_fragments/auth.py

```diff
@@ -15,20 +15,25 @@ class ModuleDocFragment(object):
   tower_host:
     description:
     - URL to your Tower or AWX instance.
+    - If value not set, will try environment variable C(TOWER_HOST) and then config files
+    default: '127.0.0.1'
```

Comment:
no, wait, I think I see what @john-westcott-iv did here, and this won't work in this form. Even if it won't work, we should document how it should work.